java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -166,42 +80,49 @@ public ColumnVector(long nativePointer) {
   public ColumnVector(DType type, long rows, Optional<Long> nullCount,
       DeviceMemoryBuffer dataBuffer, DeviceMemoryBuffer validityBuffer,
       DeviceMemoryBuffer offsetBuffer) {
+    super(ColumnVector.initViewHandle(
+        type, (int)rows, nullCount.orElse(UNKNOWN_NULL_COUNT).intValue(),
+        dataBuffer, validityBuffer, offsetBuffer, new long[] {}));
```

Comment:
Should pass `null` for the child array rather than allocating an empty array.  Last I checked, creating a view with a null child array was fine, and if it isn't we should make it so.

Reply:
done.